google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/schema/SpannerTableStatements.java

```diff
@@ -158,10 +162,22 @@ private boolean isElementCollection(Table table, Metadata metadata) {
 
     statements.add(createTableString);
 
-    // Hibernate requires the special hibernate_sequence table to be populated with an initial val.
     if (table.getName().equals(SequenceStyleGenerator.DEF_SEQUENCE_NAME)) {
-      statements.add("INSERT INTO " + SequenceStyleGenerator.DEF_SEQUENCE_NAME + " ("
-          + SequenceStyleGenerator.DEF_VALUE_COLUMN + ") VALUES(1)");
+      // Caches the INSERT statement in the RunBatchDdl auxiliary object.
+      Optional<RunBatchDdl> runBatchDdl =
```

Comment:
Maybe create a separate `findRunBatchDdl` method for clarity?

Reply:
Done.